internal-docs/prds/000-template.md

```diff
@@ -0,0 +1,39 @@
+<!--- 
```

Comment:
Adding a section in this doc about how to educate our customers of the implications of this feature would be a good idea?